scripts/install.sh

```diff
@@ -18,15 +20,20 @@ help()
 {
    echo -e "\nUsage: ENV_VAR(S) $0"\
            "--network <network>"\
-           "--skip-deployment"\
+           "--tbtc-path <tbtc-path>"\
+           "--skip-ecdsa-deployment"\
```

Comment:
Overall having just one `--skip-deployment` greatly simplifies logic in `install.sh` because of the mentioned project dependencies. The drawback is that if for some reason we need only `random-beacon` contracts, then we still need to wait until all the contracts are deployed including `tbtc`. On the other hand, this script is written to support `keep-core` client which as far as I understand requires all the contracts to be deployed to function properly.

Reply:
Exactly, the script is for the client and the client needs all the contracts. If we want to deploy just the beacon, we would do it in the `solidity/random-beacon` with `yarn deploy`.